SkillChat.Client.ViewModel/ColorGeneratorClass.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Avalonia.Media;
+using SkillChat.Client.ViewModel.Interfaces;
+
+namespace SkillChat.Client.ViewModel
+{
+    public class ColorGeneratorClass : IColorGenerator
```

Comment:
Слово Class в названии класса не нужно
```suggestion
    public class ColorGenerator : IColorGenerator
```